Sources/Turf/GeoJSON.swift

```diff
@@ -200,7 +185,7 @@ public class GeoJSON: Codable {
      
      - Returns: decoded GeoJSON of type T.
      */
-    public static func parse<T: GeoJSONObject>(_ type: T.Type, from data: Data) throws -> T {
+    public static func parse<T: GeoJSONObject>(_ type: T.Type, from data: Data) throws -> T {  // ???
```

Comment:
Replacing separate structures for each Feature with a single base one will make this method almost useless...